cps_data/finalprep.py

```diff
@@ -341,8 +341,26 @@ def adjust(data, targets):
 def benefits(data, other_ben):
     """
     Distribute benefits from non-models benefit programs and create total
-    benefits variable
+    benefits variable.
+    Replaces Medicare and Medicaid values with set amounts
     """
+    # replace medicare and medicaid
+    count_medicare = np.zeros(len(data))
+    count_medicaid = np.zeros(len(data))
+    for i in range(1, 16):
+        medicare_var = 'MCARE_VAL{}'.format(i)
+        medicaid_var = 'MCAID_VAL{}'.format(i)
+        count_medicare += np.where(data[medicare_var] > 0, 1, 0)
+        count_medicaid += np.where(data[medicaid_var] > 0, 1, 0)
+    new_medicare = count_medicare * 12000
```

Comment:
Are 12000 and 6000 necessary here? I think they just get divided away by the `scale*` columns. My understanding is that the new benefit values equal the total current benefit values divided by the number of recipients. If so you can remove the `new_medica*` columns and just use the `count`s.

If I'm missing something and they are needed, WDYT about making them constants with a brief explanation of why they're those values?

Reply:
I think Anderson is implementing my proposal, which I described in the C-TAM issue. The 12k and 6k are sort of the insurance value based on imputed benefits from MEPS, even though more precisely they should be 
<img width="722" alt="screen shot 2018-04-26 at 5 10 15 pm" src="https://user-images.githubusercontent.com/10549597/39334636-099663f4-4975-11e8-889e-6901b7dc96be.png">

I see where you're from. That would avoid the scaler step. But one caveat is that for medicare we still want to differentiate benefit by income group.

Reply:
The income upper bound refers to individual level WAS. In our case, maybe we could say if either primary or secondary earner has WAS higher than $900k, we just gave them the lower value $8776. Do we even have people with that high income in CPS tax unit dataset? If we have a good number of them, it probably worths the time to use this table. If not, I think's it's easier to go with what Max suggests. The result should be the same.

Reply:
We do have people with WAS higher than $900K. I'll switch the variables to constants, it would probably be worth differentiating between those higher earners just because there's a pretty significant difference in insurance value.